app/service.py

```diff
@@ -43,52 +43,80 @@ async def lifespan(app: FastAPI):
 )
 
 
-@app.post("/api/v1/providers", response_model=schemas.APIKey)
+@app.on_event("startup")
+async def startup():
+    await db.connect()
+
+
+@app.on_event("shutdown")
+async def shutdown():
+    await db.disconnect()
```

Comment:
The rebase wasn't successful, these methods should  no longer be here.

Reply:
sorry fixed
<img src="https://i.pinimg.com/originals/b9/b5/16/b9b516d3a21e5d440b88f2caa9a22c6d.gif"/>

Reply:
@tchx84 👀

Reply:
<img src="https://www.memecreator.org/static/images/memes/3768166.jpg" />